src/app/profile/me/page.tsx

```diff
@@ -1,54 +1,89 @@
 'use client';
 
-import { Box, VStack } from '@chakra-ui/react';
-import Link from 'next/link';
-import { usePathname, useRouter } from 'next/navigation';
-
-import AuthRequired from '@/ui/AuthRequired';
-import ProfileInfo from '@/ui/Profile/ProfileInfo';
-import ProfileBookShelf from '@/ui/Profile/ProfileBookshelf';
-import ProfileGroup from '@/ui/Profile/ProfileGroup';
-import Button from '@/ui/common/Button';
-import { Menu, MenuItem } from '@/ui/common/Menu';
-import { removeAuth } from '@/utils/helpers';
+import { useRouter } from 'next/navigation';
+import { isAuthed, removeAuth } from '@/utils/helpers';
 import userAPI from '@/apis/user';
+import TopHeader from '@/v1/base/TopHeader';
+import ProfileInfo from '@/v1/profile/info/ProfileInfo';
+import ProfileBookShelf from '@/v1/profile/bookShelf/ProfileBookShelf';
+import ProfileGroup from '@/v1/profile/group/ProfileGroup';
+import Avatar from '@/v1/base/Avatar';
+import Link from 'next/link';
+import { IconArrowRight } from '@public/icons';
+import BookShelf from '@/v1/bookShelf/BookShelf';
+
+const USER_ID = 'me';
+const KAKAO_LOGIN_URL = `${process.env.NEXT_PUBLIC_API_URL}/oauth2/authorize/kakao?redirect_uri=${process.env.NEXT_PUBLIC_CLIENT_REDIRECT_URI}`;
 
 const MyProfilePage = () => {
-  const { push } = useRouter();
-  const pathname = usePathname();
+  return isAuthed() ? <MyProfileForAuth /> : <MyProfileForUnAuth />;
+};
+
+const MyProfileForUnAuth = () => {
+  return (
+    <>
+      <TopHeader text="Profile" />
+      <div className="flex flex-col gap-[2rem]">
+        <div className="flex flex-col gap-[2rem]">
+          <div className="flex items-center gap-[1rem]">
+            <Avatar size="large" />
+            <div className="flex-grow">
+              <h2 className="text-lg font-bold">로그인 / 회원가입</h2>
+              <p className="text-placeholder">
+                카카오로 3초만에 가입할 수 있어요.
+              </p>
+            </div>
+            <Link href={KAKAO_LOGIN_URL}>
+              <IconArrowRight width="20px" />
+            </Link>
+          </div>
```

Comment:
p2;

아래 `ProfileInfoPresenter` 파일에서도 언급을 했었는데, 프로필 정보와 책장/참여한 모임 UI가 좀 더 구분될 수 있도록 2rem의 여백이 추가되면 좋을 것 같아요! 그리고 '카카오로 3초안에 가입할 수 있어요.' 텍스트의 스타일에 `text-sm`도 추가해주세요!! 🙇🏻‍♀️ 

```suggestion
          <div className="flex items-center gap-[1rem] mb-[2rem]">
            <Avatar size="large" />
            <div className="flex-grow">
              <h2 className="text-lg font-bold">로그인 / 회원가입</h2>
              <p className="text-placeholder text-sm">
                카카오로 3초만에 가입할 수 있어요.
              </p>
            </div>
            <Link href={KAKAO_LOGIN_URL}>
              <IconArrowRight width="20px" />
            </Link>
          </div>
```

Reply:
@gxxrxn https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/pull/461/commits/0279838b5743bdd33b56832f26a23972b5408d57